pkg/api/handlers/compat/resize.go

```diff
@@ -71,14 +62,6 @@ func ResizeTTY(w http.ResponseWriter, r *http.Request) {
 			utils.SessionNotFound(w, name, err)
 			return
 		}
-		if state, err := ctnr.State(); err != nil {
```

Comment:
I think this bit is still reasonable to hit - exec sessions in a non-running container will never happen.

Reply:
I added this back. It looks `podman exec -it con1 stty size` is broken as well but for both local and remote.